### PR TITLE
Fix Admin Router service endpoint permission ref

### PR DESCRIPTION
**Pull request by timaa2k**
## Urgency
- [ ] Blocker 
- [x] High
- [ ] Medium

## Requirements
- Test all commands and procedures.
- Add [redirects](https://github.com/mesosphere/dcos-docs-site/wiki/Redirects).
- Change all affected versions (e.g. 1.7, 1.8, 1.9, 1.10, 1.11, 1.12).
- See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing).

### CODE REVIEW

File: pages/1.11/security/ent/perms-reference/index.md [hunk 1]
```diff
@@ -82,7 +82,7 @@ through Admin Router.
...
 | `dcos:adminrouter:secrets`<br> Controls access to [Secrets](/1.11/security/ent/secrets/) web interface and API.                                                                                                                                                                              | x    |   |   |   |   |
-| `dcos:adminrouter:service[:<group-name>]/<service-name>`<br> Controls access to the web interface and API of an installed DC/OS service.                                                                                                                                                        | x    |   |   |   |   |
+| `dcos:adminrouter:service:[<group-name>]/<service-name>`<br> Controls access to the web interface and API of an installed DC/OS service.                                                                                                                                                        | x    |   |   |   |   |
```
Comment (jgehrcke) [inline pages/1.11/security/ent/perms-reference/index.md]: If we want to make this more correct then the slash also needs to be in the squared brackets, no?

Comment (jgehrcke) [reply pages/1.11/security/ent/perms-reference/index.md]: These are valid examples:
```
dcos:adminrouter:service:org/dev/apps/svc1
```

```
dcos:adminrouter:service:svc1
```

Comment (jgehrcke) [reply pages/1.11/security/ent/perms-reference/index.md]: This is not a valid example:
```
dcos:adminrouter:service:/svc1
```

Comment (timaa2k) [reply pages/1.11/security/ent/perms-reference/index.md]: True

**Comment by timaa2k** [comment]
@jgehrcke please approve if you're happy!

**Comment by timaa2k** [comment]
I've added the `documentation` label on the JIRA ticket.
@ck4adventure the docs ticket is now available under: [DCOS-52157](https://jira.mesosphere.com/browse/DCOS-52157)
Is there anything else I can do to push this forward?

**Comment by ck4adventure** [comment]
@timaa2k et al: Usually docs team gets notified through the jira system and plans/sprints this work accordingly, as we have to balance incoming requests from all teams. That is why a docs ticket is needed. I will get this to @jpe442  to be assigned and we will try to get to it as soon as we can.

**Pull request merged**